package.json

```diff
@@ -40,7 +40,6 @@
     "devDependencies": {
         "@types/jest": "^24.0.13",
         "@types/react": "^16.8.0",
-        "@types/react-redux": "^7.1.0",
```

Comment:
why did you remove that?

Reply:
It was a very hot day and my melted brain mistaken it for redux-form types...